packages/kbn-securitysolution-list-utils/src/build_exception_filter/index.ts

```diff
@@ -290,6 +292,25 @@ export const buildMatchAnyClause = (entry: EntryMatchAny): BooleanFilter => {
   }
 };
 
+export const buildMatchWildcardClause = (entry: EntryMatchWildcard): BooleanFilter => {
```

Comment:
Can we add a unit test for this like the others here - `x-pack/plugins/lists/common/exceptions/build_exceptions_filter.test.ts `?

Reply:
done!